control-agent/text-to-speech/realsense.py

```diff
@@ -0,0 +1,49 @@
+import pyrealsense2 as rs
+import numpy as np
+import time
+import math
+
+from PIL import Image
+import io
+
+import speech
+
+
+def take_photo():
```

Comment:
Consider breaking this up into smaller functions

Reply:
adding to refactor issue